Assets/MixedRealityToolkit.Services/InputAnimation/IMixedRealityInputRecordingService.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    /// <summary>
+    /// Provides input recording into an internal buffer and exporting to files.
+    /// </summary>
+    public interface IMixedRealityInputRecordingService : IMixedRealityInputDeviceManager
+    {
+        /// <summary>
+        /// Input is being recorded.
+        /// </summary>
+        bool IsRecording { get; }
+
+        /// <summary>
+        /// Limit the size of the recording buffer.
+        /// </summary>
+        /// <remarks>
+        /// If recording is limited any input older than the RecordingBufferTimeLimit will be discarded.
+        /// </remarks>
+        bool UseBufferTimeLimit { get; set; }
+
+        /// <summary>
+        /// Size of the input recording buffer.
+        /// </summary>
+        float RecordingBufferTimeLimit { get; set; }
```

Comment:
It would also be helpful to add some clarification here about the units - i.e. the comments say "size of input recording buffer" but the variable name is "time limit", which is a bit add odds with each other (i.e. maybe it's size in seconds? or really, duration, in seconds? Or minutes, or milliseconds, etc)

Reply:
Added explanation in comments.